app/src/main/java/com/artemzin/qualitymatters/models/NetworkBitmapClient.java

```diff
@@ -0,0 +1,7 @@
+package com.artemzin.qualitymatters.models;
+
+import android.widget.ImageView;
+
+public interface NetworkBitmapClient {
```

Comment:
Maybe rename to something like: `QualityMattersImageLoader`?

I don't have strong opinion here